crates/autopilot/src/database/ethflow_events/event_retriever.rs

```diff
@@ -28,6 +30,7 @@ impl EthFlowRefundRetriever {
 impl EventRetrieving for EthFlowRefundRetriever {
     type Event = contracts::cowswap_eth_flow::Event;
 
+    #[instrument(skip_all)]
```

Comment:
This function itself doesn't really do much. Similar to an async function that's not getting polled.
Since it's only called once it doesn't matter too much if we have the instrumentation or not.
Same comment applies to the other functions that return a `EvensBuilder`.